corehq/apps/users/templates/users/web_users.html

```diff
@@ -1,365 +1,697 @@
-{% extends "users/base_template.html" %}
+{% extends 'style/bootstrap3/base_section.html' %}
+{% load crispy_forms_tags %}
 {% load i18n %}
 {% load hq_shared_tags %}
+{% load djangular_tags %}
 
 {% block js %}{{ block.super }}
     <script src="{% static 'hqwebapp/js/main.js' %}"></script>
-    <script src="{% static 'style/lib/knockout_plugins/knockout_mapping.ko.min.js' %}"></script>
-    <script src="{% static 'style/ko/knockout_bindings.ko.js' %}"></script>
     <script src="{% static 'users/js/roles.js' %}"></script>
+    <script src="{% static 'users/js/web_users_list.ng.js' %}"></script>
```

Comment:
Still perplexing is why this worked on India...

On Thursday, September 10, 2015, Ben Rudolph <brudolph@dimagi.com> wrote:

> Good catch
>
> On Thursday, September 10, 2015, Biyeun <notifications@github.com
> <javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:
>
>> In corehq/apps/users/templates/users/web_users.html
>> <https://github.com/dimagi/commcare-hq/pull/7499#discussion_r39230992>:
>>
>> >      <script src="{% static 'users/js/roles.js' %}"></script>
>> > +    <script src="{% static 'users/js/web_users_list.ng.js' %}"></script>
>>
>> Yes it should be new_static
>> … <#14fb9f013636d2b9_14fb9ce0e9044bb6_>
>> On Thu, Sep 10, 2015 at 18:48 Cory Zue <notifications@github.com> wrote:
>> In corehq/apps/users/templates/users/web_users.html <
>> https://github.com/dimagi/commcare-hq/pull/7499#discussion_r39224035>: >
>> <script src="{% static 'users/js/roles.js' %}"></script> > + <script
>> src="{% static 'users/js/web_users_list.ng.js' %}"></script> @orangejenny
>> <https://github.com/orangejenny> <https://github.com/orangejenny> @biyeun
>> <https://github.com/biyeun> <https://github.com/biyeun> should this be
>> new_static instead of static? we're trying to figure out the root cause of
>> this bug http://manage.dimagi.com/default.asp?181567 — Reply to this
>> email directly or view it on GitHub <
>> https://github.com/dimagi/commcare-hq/pull/7499/files#r39224035>.
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/dimagi/commcare-hq/pull/7499/files#r39230992>.
>>
>